docs/deploying-java-app-with-database.md

```diff
@@ -0,0 +1,290 @@
+    $ odo url create --host $(minikube ip).nip.io
+    ```
+
+5.  Push the URL to activate it:
+    
+    ``` sh
+    $ odo push
+    ```
+
+6.  Display the created URL:
+    
+    ``` sh
+    $ odo url list
+    ```
+    
+    You will see a fully formed URL that can be used in a web browser:
+    
+    ``` sh
+    [root@pappuses1 jpa]# odo url list
+    Found the following URLs for component mysboproj
+    NAME               STATE      URL                                           PORT     SECURE     KIND
+    mysboproj-9080     Pushed     http://mysboproj-9080.192.168.49.2.nip.io     9080     false      ingress
+    [root@pappuses1 jpa]#
+    ```
+
+7.  Use the URL to navigate to the `CreatePerson.xhtml` data entry page and enter requested data: URL/CreatePerson.xhtml' and enter a user’s name and age data using the the form.
+
+8.  Click on the **Save** button when complete
+
+Note that the entry of any data does not result in the data being displayed when you click on the "View Persons Record List" link.
+
+### Binding the database and the application
+
+Now, the only thing that remains is to connect the DB and the application. We will use odo to create a link to the Dev4Devs PostgreSQL Database Operator in order to access the database connection information.
+
+1.  Display the services available to odo: - You will see an entry for the PostgreSQL Database Operator displayed:
+    
+    ``` sh
+    $ odo catalog list services
+    Operators available in the cluster
+    NAME                                             CRDs
+    postgresql-operator.v0.1.1                       Backup, Database
+    ```
+
+2.  List the service associated with the database created via the PostgreSQL Operator:
+    
+    ``` sh
+    $ odo service list
+    NAME                        AGE
+    Database/sampledatabase     6m31s
+    ```
+
+3.  Create a Service Binding Request between the application and the database using the Service Binding Operator service created in the previous step `odo link` command:
+    
+    ``` sh
+    $ odo link Database/sampledatabase
+    ```
+
+4.  Push this link to the cluster:
+    
+    ``` sh
+    $ odo push
+    ```
+    
+    After the link has been created and pushed a secret will have been created containing the database connection data that the application requires.
+    
+    You can inspect the new intermediate secret via the dashboard console in the 'my-postgresql-operator-dev4devs-com' namespace by navigating to Secrets and clicking on the secret named `mysboproj-database-sampledatabase` Notice it contains 4 pieces of data all related to the connection information for your PostgreSQL database instance.
+
+5.  Push the newly created link. This will terminate the existing application pod and start a new application pod.
+    
+    ``` sh
+    odo push
+    ```
+
+6.  Once the new pod has initialized you can see the secret database connection data as it is injected into the pod environment by executing the following:
```

Comment:
I think adding a comma after "Once the new pod has initialized" will make more sense.